### ISSUE TITLE
Enhancements: Base point cost in printable army list

### ISSUE THREAD

**Issue by Viper666-Qc** [BSData/wh40k-7th-edition#367]
Could it be possible to list the base point cost of every unit somewhere on the army list (base points before any  upgrade) and make it available to see in the printable list?  It would be far more easy to catch point cost errors vs the codex

**Comment by Kangodo** [comment]
That depends on how we format the file.
In Blood Angels I have an entry called Death Company and a separate one for weapons.
It shows: "8x Death Company Marine (160pts)" and "4x Power Weapon (60pts).
But in the Grey Knight Strike Squad there are entries that show as "2x Grey Knight with hammer (60pts)".

It really depends on how you write the unit, because it's different for each unit and each author.
Having those Grey Knights as a basic with a separate weapon-entry is inefficient and impossible to do in a correct way.
The weapons used to be separate, but then it didn't correctly replace the old weapons.

So in short: No, that is not possible for many units unless we redo everything which causes other (more important) issues.

**Comment by Viper666-Qc** [comment]
Shouldn't you "standardize" between authors? It will become more complicated as more people seem to contribute lately, in particular if each author do it his way...

**Comment by amis92** [comment]
I'd love to answer: _That's a great idea! We'll do it!_

Unfortunately me being more realistic turn it into: _Oh, we'd love to, but it's quite too hard probably_.

See, those are reasons:
1. We'd love to have a single scheme. We really would.
2. There's a huge amount of work connected with it, tough.
3. And we do it in free time, from the deepest parts of our hearts and souls. As best as we can.
4. And we have in disposal only what BattleScribe gives us. Which means many tools, but not unlimited, like scripts in ArmyBuilder.

What would need to happen to standardize:
1. We'd need to write some standard. And that's currently started/in progress/an idea was seeded. But it didn't catch huge interest and no-one really is involved in it.
2. We'd need to rewrite catalogues unto this new standard.
3. We can't have a standard, in fact. The problem is GW always catches us with some unexpected limits on weapons, weird conditions and large and wide crossover limits, like, I don't know, a Wolf Guard can take Plasma only if allied detachment contains Captain Shrike (joking).
4. As **Kangodo** mentioned, there are cases which although very similar, unit layouts differ because it reads better or because there is no real way

But we'll do our best.

**Comment by ghost** [comment]
I agree with @amis92 - in fact most of my work has been modelled on his Tau file - and I'd add that over time we learn different ways of doing things, so the new IG file looks & works much better than the old Vraks catalogues, for example.

**Comment by Kangodo** [comment]
I agree too and I think the biggest argument against 'standardisation' is that GW isn't being consistent and standard in their Codices.
Just look at how the 'Assault Squads' from DA, CSM, BA and GK are different in the Codex.

**What we always aim to do is to create something that is easy to use in building a roster.**
In my opinion that rules out anything that comes down to units like the GK-Paladins, though that would be the most correct way to handle any unit.
I can't even begin to imagine the stress of creating a unit of 30 Death Companies, model by model.
The only reason Paladins are like that, is because there is no other choice.

A lot of this comes down to: **What does the author and the players prefer?**
Just for this I've created a thread on B&C-BA to get feedback and people are quite happy with how the current catalogue works, even though there are _a lot_ of errors.
An example of such an error would be the fact that the ASM don't have a Bolt Pistol and Chainsword in their entry. But adding those would be quite annoying for me and actually decrease the ease of list-building for end-users, so that is not something I want to do.

It would be nice to have a standard - even if it were possible - but you also have to consider the authors themselves!
One might like it if things are 100% correct while another might prefer in his catalogue the ease of building a list within 5 minutes.
I wouldn't like it if I had to built "my codex" in a way I don't like and I think the same goes for many others. 
_Doing things the way the author prefers it is a privilege all these hard workers deserve._
